### DISCUSSION TITLE
Getting this error while exporting the project "The Security Manager is deprecated and will be removed in a future release"

### DISCUSSION THREAD

**Discussion by yogesh-7** [varabyte/kobweb#374] [Q&A]
![Screenshot 2023-11-11 at 2 47 47 PM](https://github.com/varabyte/kobweb/assets/54579951/cea727a0-8cf4-4707-a425-924e9a4d009c)

**Comment by bitspittle** [comment]
What version of Kobweb are you using?

At the very least, based at what I'm reading online, it seems like you can use an older version of Java to compile your project to work around this for now. Are you using Java 17? 19?

The Kobweb codebase itself does not use SecurityManager, directly at least, so if you have a more relevant callstack to share, that would be helpful. 

Can you scroll down a bit? There should eventually be the real core exception, where the callstack lines will be my code, and not an endless list of `org.gradle.blah` stuff. Thanks!

**Reply by bitspittle** [reply]
By the way, this issue sounded familiar, and I remember I did used to use SecurityManager, but I removed it 6 months ago: https://github.com/varabyte/kobweb/commit/92262f3d774adb511576108a1281f923f1dd0be4

**Comment by yogesh-7** [comment]
Thank you for reaching out! Regarding your question about the version of Kobweb we are using, currently, we are on version Kobweb 0.9.13.

**Reply by bitspittle** [reply]
Ah sorry, not the CLI, but the library. Check your libs.version.toml file.

https://github.com/varabyte/kobweb#gradle-version-catalogs
